metadata-ingestion/src/datahub/ingestion/source/dbt/dbt_cloud.py

```diff
@@ -32,10 +33,17 @@
 
 
 class DBTCloudConfig(DBTCommonConfig):
+    access_url: str = Field(
+        description="The base URL of the dbt Cloud instance to use. This should be the URL you use to access the dbt Cloud UI. It should include the scheme (http/https) and not include a trailing slash. See access url for you dbt Cloud region here: https://docs.getdbt.com/docs/cloud/about-cloud/regions-ip-addresses",
```

Comment:
```suggestion
        description="The base URL of the dbt Cloud instance to use. This should be the URL you use to access the dbt Cloud UI. It should include the scheme (http/https) and not include a trailing slash. See the access url for your dbt Cloud region here: https://docs.getdbt.com/docs/cloud/about-cloud/regions-ip-addresses",
```
Can we also allow trailing slash by preprocessing this, e.g. add a validator that does `access_url = access_url.rstrip("/")`